docs/tutoriales/como-mintear.mdx

```diff
@@ -1,80 +1,121 @@
 ---
 id: como-mintear
-title: ¿Cómo mintear NFTs?
-sidebar_label: ¿Cómo mintear NFTs?
+title: ¿Cómo acuñar NFTs?
+sidebar_label: ¿Cómo acuñar NFTs?
```

Comment:
I don't think `acuñar` is the best translation

Reply:
Leo told me to change `mintear` for `acuñar` so i don't really know what word fits best

Reply:
![image](https://user-images.githubusercontent.com/71234240/154094957-2a09ebe0-cbb9-4885-b3fa-cd2db3303f10.png)

Reply:
Perfect